### PR TITLE
Style results

### LINKED ISSUE #39: Add source method component to results container

### PR DESCRIPTION
**Pull request by VeeAndrade**
### Sidebar Checklist
​
- [x] Ran test suite
- [x] Request reviewers
- [x] Assign yourself and other contributors
- [x] Link to project
- [ ] Added to Readme
​
### Issues Resolved
Resolves #39 
​
### Problem Addressed
​Needed to have some sort of display of the method that the user wanted to get translated
### Solution Implemented
​Created a source method component inside the results container
### Screenshots (optional)
Screen Shot 2020-04-14 at 10 40 05 AM

### CODE REVIEW

File: src/components/SourceMethod/SourceMethod.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+          <div className='syntax-div'>
+          <p>Syntax: Array.from(arrayLike [, mapFn [, thisArg]])\n</p>
+          </div>
+      </section>
+      <section className='method-bottom-sect'>
+        <div>
+          <p>Snippet: console.log(Array.from('foo'));
+            \n// expected output: Array [\"f\", \"o\", \"o\"]\n\n
+            console.log(Array.from([1, 2, 3], x => x + x));\n// expected
+             output: Array [2, 4, 6]\n</p>
```
Comment (DanielEFrampton) [inline src/components/SourceMethod/SourceMethod.jsx:23]: I assume this is a placeholder for what Ali and Josh are going to get populated from the query?

Comment (mintona) [reply src/components/SourceMethod/SourceMethod.jsx:23]: Correct!

**Review by DanielEFrampton — APPROVED** [review]
Awesome!

**Review by mintona — APPROVED** [review]
Lookin good! Ready to add dynamic source methods on the next branch.

**Pull request merged**